hack/test-e2e.sh

```diff
@@ -16,71 +19,78 @@ while ! nc -vz localhost 8080 > /dev/null 2>&1 ; do
     sleep 0.1
 done
 
-# Create a Stake Holder Group
-curl -X POST ${host}/stakeholdergroups -d \
-'{
-    "name": "Big Dogs",
-    "description": "Group of big dogs."
-}' | jq -M .
+# Create a Stake Holder Group if one does not exist
+stakeholder_groups="$(curl -X GET ${host}/stakeholdergroups)"
+if [[ "${stakeholder_groups}" == "[]" ]]; then
```

Comment:
What you are doing here is testing if there are any stakeholder groups when we need to test if the specific stakeholder group we are about to create exists.